Documentation/subcommands/config.md

```diff
@@ -0,0 +1,46 @@
+# rkt config
+
+This command prints the rkt configuration per stage in JSON format.
+
+## Example
+
+```
+$ rkt config
+{
+  "stage0": [
```

Comment:
OOB notes:
- define top-level struct, as this is new
- explain why we are printing each entry separately (even if the input config was deduplicated)
- explain oauth/basic mutual incompatibility and override semantics